lib/sequin/accounts/accounts.ex

```diff
@@ -736,4 +736,66 @@ defmodule Sequin.Accounts do
 
     update_account(account, %{features: features})
   end
+
+  def invite_user(%User{} = user, %Account{} = account, sent_to, url_fun) when is_function(url_fun, 1) do
```

Comment:
```suggestion
  def invite_user(%User{} = inviting_user, %Account{} = account, send_to, url_fun) when is_function(url_fun, 1) do
```
I think these names are a bit more clear that the user is inviting to an email (which we have not yet sent to)